activestorage/test/controllers/representations/proxy_controller_test.rb

```diff
@@ -37,6 +38,20 @@ class ActiveStorage::Representations::ProxyControllerWithVariantsTest < ActionDi
     assert_equal 67, image.height
   end
 
+  test "showing untracked variant" do
```

Comment:
as expected when variant tracking is disabled, there is an `NoMethodError: undefined method `forced_disposition_for_serving' for #<ActiveStorage::Variant:0x0000000108838110`

![image](https://github.com/rails/rails/assets/1371733/5fc6dd6e-e863-45fe-8b7d-45353e76f125)

Reply:
done in #50107